toolkit/templates/forms/keyword-search.html

```diff
@@ -1,8 +1,11 @@
-<div class="govuk-option-select">
+<div class="govuk-option-select filter-field-text">
   <div class="container-head">
     <label class="option-select-label" for="{{ id }}">
       {{ label }}
     </label>
   </div>
-  <input class="filter-field-text" type="text" name="{{ name }}" id="{{ id }}" value="{{ value }}" maxlength="{% if maxLength %}{{ maxLength }}{% else %}200{% endif %}">
+  <div class="container-search">
+    <input type="search" name="{{ name }}" id="{{ id }}" type="{{ label }}" value="{{ value }}" maxlength="{% if maxLength %}{{ maxLength }}{% else %}200{% endif %}">
```

Comment:
Oh no! Final point! (I thought I removed this but I guess it didn't stick!)

There are two `type` attributes.  We want to keep the one that says type="search" and not the other one.